main.cpp

```diff
@@ -124,6 +140,17 @@ struct CoffeeMachine
     float beansLevel, timeSinceCleaning;
     std::string coffeeBeans;
     int cupsPerRun { 2 };
+    float refillReminder(float wtLvl = 100.f, float wtLimit = 15.f)
+    {
+        wtLvl = waterLevel;
```

Comment:
You are overwriting your function parameter's value with your member variable's value.
Are you sure you want to do that?
- [ ] revise